specs/spring-cloud-contract-spec-java/src/main/java/org/springframework/cloud/contract/spec/internal/OptionalProperty.java

```diff
@@ -44,8 +44,15 @@ public String optionalPattern() {
 	}
 
 	public String value() {
-		return this.value instanceof RegexProperty
-				? ((RegexProperty) this.value).pattern() : this.value.toString();
+		if (this.value == null) {
+			return "";
```

Comment:
Not sure if this is what needs to be returned here...

Reply:
There should be no such option that `null` is passed in. We can leave `""`